tests/thingpedia-integration.sh

```diff
@@ -34,7 +34,7 @@ mkdir -p $workdir/etc/config.d
 export THINGENGINE_CONFIGDIR=$workdir/etc
 PORT=${PORT:-8080}
 cat > ${THINGENGINE_CONFIGDIR}/config.d/99-local.yaml <<EOF
-DATABASE_URL: "mysql://thingengine:thingengine@localhost/thingengine_test"
+DATABASE_URL: "mysql://thingengine:thingengine@localhost/thingengine"
```

Comment:
Please do not change this. The test database should be called `thingengine_test`, to distinguish it from the real database.

Reply:
Sorry, didn't mean to commit this. Changed back.